tools/query/.shed.yml

```diff
@@ -0,0 +1,6 @@
+name: query

```

Comment:
the name is very generic, this will be super difficult to search for

Reply:
Since we have started the more serious discussion with all of you involved, I propose to contribute this working prototype to a similar existing IUC tool [Query Tabular](https://github.com/galaxyproject/tools-iuc/blob/master/tools/query_tabular/query_tabular.xml) which works only on text tabular files. Otherwise, I agree that the name is too generic.

Reply:
Uh, I :heart:  that!